supplementary_style_guide/introduction/about-guide.adoc

```diff
@@ -4,11 +4,10 @@ This guide provides style guidelines and preferred term usage for link:https://w
 
 The Red Hat Customer Content Services team created this guide as part of its responsibility for producing product documentation for Red Hat customers. Upstream communities who want to align more closely with the standards used by Red Hat product documentation can also use this guide. 
 
-[NOTE]
-====
-Both the link:https://www.ibm.com/docs/en/ibm-style[IBM Style Guide] and the link:https://github.com/redhat-documentation/supplementary-style-guide/issues[Red Hat supplementary style guide] are the source of truth for output and writing style guidance that is not specific to Modular Documentation guidelines. The link:https://redhat-documentation.github.io/modular-docs/[Modular Documentation Reference Guide] and templates are the source of truth for all things connected to Modular Documentation, including implementing those guidelines in Asciidoc. The link:https://redhat-documentation.github.io/asciidoc-markup-conventions/[AsciiDoc Mark-up Quick Reference] is the source of truth for guidance specific to writing in Asciidoc.
-====
-
 == How to use this guide
 
 When looking for style guidance, reference this guide first, because it overrides certain guidance from the _IBM Style Guide_. If you must deviate from the guidance in either guide, notify the style council by opening an link:https://github.com/redhat-documentation/doc-style/issues[issue]. This way, the deviation can be discussed by the style council and included in this guide.
+
+== Other style guides
```

Comment:
@mportman12 This is kind of wishy-washy. If the point of this PR is to provide the guidance on the source of truth, then don't be shy about saying that here. This is the guidance we want to provide here, so I think the heading should be:
`== Source of truth`

@ndeevy @bergerhoffer @cbudz @inoxx03 Curious to know what others think.

Reply:
I understand that we want to clarify the source of truth, but I don't think that we necessarily need to use that phrasing.

As for the existing wording, I wouldn't necessarily call them style guides, because that might get confusing (because the SSG is the style guide). 

"Related Red Hat product documentation guides" or "Official Red Hat ..." or something along that lines.